tests/sgl/func/test_tensor.cpp

```
@@ -0,0 +1,146 @@
+namespace {
+
+struct TensorTestType {
+    ref<refl::Type> dtype;
+    ref<TypeLayoutReflection> element_layout;
+};
+
+TensorTestType make_tensor_test_type(Device* device)
+{
+    ref<SlangModule> module = device->load_module_from_source(
+        "func_tensor_tests",
+        R"(
```

Cppcheck (2.20.0)

[performance] Variable 'm_logger' is assigned in constructor body. Consider performing initialization in initialization list.

(useInitializationList)